Kickstarter-iOS/Views/Controllers/SettingsViewController.swift

```diff
@@ -827,6 +833,10 @@ internal final class SettingsViewController: UIViewController {
     self.helpViewModel.inputs.helpTypeButtonTapped(.privacy)
   }
 
+  @IBAction func privateProfileSwitchDidChange(_ sender: UISwitch) {
```

Comment:
does this need to be `fileprivate` also?

Reply:
Added!